calculate_movement_wrangler.py

```diff
@@ -189,14 +193,19 @@ def lambda_handler(event, context):
 
             logger.info("Successfully filtered and merged the previous period data")
 
+            anomalies = []
+
             # Pass to mismatch detector to look for and fix strata mismatches
-            merged_data, anomalies = strata_mismatch_detector(merged_data, period, time,
-                                                              reference, segmentation,
-                                                              stored_segmentation,
-                                                              current_time,
-                                                              previous_time,
-                                                              current_segmentation,
-                                                              previous_segmentation)
+            if "strata" in distinct_values:
+                merged_data, anomalies = strata_mismatch_detector(
+                    merged_data,
+                    period, time,
+                    reference, segmentation,
+                    stored_segmentation,
+                    current_time,
+                    previous_time,
+                    current_segmentation,
+                    previous_segmentation)
 
             logger.info("Successfully completed strata mismatch detection")

```

Comment:
Should this be indented to be inside the `if` to only fire if strata_mismatch_detector was actually run?

Reply:
Fixed.